e2e/cypress/integration/archived_channel/archived_leave_channel_spec.js

```diff
@@ -28,303 +24,169 @@ describe('Leave an archived channel', () => {
             },
         });
 
-        // # Login as test user and visit town-square
-        cy.apiInitSetup().then(({team, channel, user}) => {
+        // # Login as test user and visit test channel
+        cy.apiInitSetup({
+            loginAfter: true,
+            promoteNewUserAsAdmin: true,
+        }).then(({team, user}) => {
             testTeam = team;
-            testChannel = channel;
             testUser = user;
 
             cy.apiCreateUser({prefix: 'second'}).then(({user: second}) => {
                 cy.apiAddUserToTeam(testTeam.id, second.id);
                 otherUser = second;
             });
-            cy.visit(`/${team.name}/channels/${testChannel.name}`);
-            cy.postMessageAs({sender: testUser, message: testArchivedMessage, channelId: testChannel.id});
-        });
-    });
-
-    it('should leave recently archived channel', () => {
```

Comment:
Deleted since [MM-T1685](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T1685) was already automated at [here](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/integration/archived_channel/leave_archived_channel_spec.js#L28).